mtp_noms_ops/templates/settings/job-information.html

```diff
@@ -0,0 +1,97 @@
+{% extends 'base.html' %}
```

Comment:
HTML, CSS and JS use a 2-space indent style and i think it's worth indenting these files the same as the rest of the app